packages/node_modules/@webex/internal-plugin-mercury/src/mercury.js

```diff
@@ -3,10 +3,13 @@
  */
 
 import url from 'url';
+
 import {WebexPlugin} from '@webex/webex-core';
 import {deprecated, oneFlight} from '@webex/common';
 import {camelCase, get, set} from 'lodash';
 import backoff from 'backoff';
+import HttpsProxyAgent from 'https-proxy-agent';
+
```

Comment:
I didn't re-order these, I assume something is changing them as part of the commit process?